docs/guides/intersphinx.rst

```diff
@@ -0,0 +1,143 @@
+Link to Other Project's Documentation With Intersphinx
+======================================================
+
+You may be familiar with using the :ref:`:ref: role <sphinx:ref-role>` to link to any location of your docs.
+It helps you to keep all links up to date, Sphinx will raise a warning if the reference doesn't exist.
+
+Some times you may need to link to a location of another documentation project.
+We could just link to where the documentation is hosted,
+and use Sphinx's ``linkcheck`` builder to check for broken links.
```

Comment:
Sphinx doesn't have any example on their docs to link to, but this usually a target in their makefile `make linkcheck`

Reply:
And not sure if is worth mentioning here, but I wanted to make clear that users can still have that feature from :ref: without using intersphinx